tests/unit_test/action/action_test.py

```
@@ -1055,6 +1056,288 @@ async def test_run_no_http_action(self):
+        dispatcher: CollectingDispatcher = CollectingDispatcher()
+        tracker = Tracker(sender_id="sender1", slots=slots, events=events, paused=False, latest_message=latest_message,
+                          followup_action=None, active_loop=None, latest_action_name=None)
+        domain: Dict[Text, Any] = None
+        action_call = {
+            "next_action": action_name,
+            "sender_id": "sender1",
+            "tracker": tracker,
+            "version": "3.6.21",
+            "domain": domain
+        }
+        actual: List[Dict[Text, Any]] = await ActionProcessor.process_action(dispatcher, tracker, domain, action_name,action_call = action_call)
```

Ruff (0.11.9)

Local variable `actual` is assigned to but never used

Remove assignment to unused variable `actual`

(F841)